ta4j-core/src/main/java/org/ta4j/core/bars/BaseBarBuilder.java

```diff
@@ -50,10 +50,16 @@ public class BaseBarBuilder implements BarBuilder {
     private long trades;
     private BarSeries baseBarSeries;
 
+    /** A builder to build a new {@link BaseBar} with {@link DoubleNumFactory} */
     public BaseBarBuilder() {
         this(DoubleNumFactory.getInstance());
     }
 
+    /**
+     * Constructor.
```

Comment:
I am not a fan of documentation that does not tell anything. 

I know it is constructor by language syntax. 
I know it takes numFactory.

There is no added value, just boiler plate doc.

Reply:
I vote for deletion.

Reply:
If a comment is deemed by all parties to be unnecessary within this change set, it should be removed. That the pattern exists elsewhere in the codebase should not prevent said removal, otherwise we are intentionally adding tech debt.